apps/communications/contacts/js/activities_vcard.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+/* global ActivityHandler, LazyLoader, VCardReader, Contacts */
+
+ActivityHandler.vCardActivityHandler = {
```

Comment:
instead of this being an object this should be a closure and as a result you can keep private variables, const and the like